roles/openshift_on_openstack/templates/OSEv3.yml.cfg.j2

```diff
@@ -25,10 +25,14 @@ openshift_storage_glusterfs_namespace: glusterfs
 openshift_storage_glusterfs_is_native: true
 openshift_storage_glusterfs_storageclass_default: true
 openshift_storage_glusterfs_block_storageclass: true
-openshift_storage_glusterfs_block_version: v{{ ocp_major_minor }}
-openshift_storage_glusterfs_version: v{{ ocp_major_minor }}
-openshift_storage_glusterfs_heketi_version: v{{ ocp_major_minor }}
-openshift_storage_glusterfs_s3_version: v{{ ocp_major_minor }}
+#openshift_storage_glusterfs_block_version: v{{ ocp_major_minor }}
+#openshift_storage_glusterfs_version: v{{ ocp_major_minor }}
+#openshift_storage_glusterfs_heketi_version: v{{ ocp_major_minor }}
+#openshift_storage_glusterfs_s3_version: v{{ ocp_major_minor }}
+openshift_storage_glusterfs_block_version: v3.9
```

Comment:
I am guessing this is temporary until the 3.10 gluster containers start working?

Reply:
Yes.